specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2940 @@
+          "readOnly": true
+        },
+        "bootType": {
+          "type": "string",
+          "description": "Boot type of the machine.",
+          "enum": [
+            "Unknown",
+            "EFI",
+            "BIOS"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "MachineBootType",
+            "modelAsString": true
+          }
+        },
+        "datacenterContainer": {
+          "type": "string",
+          "description": "Container defined in the management solution that this machine is part of in the datacenter.",
+          "readOnly": true
+        },
+        "datacenterManagementServer": {
+          "type": "string",
+          "description": "Name of the server hosting the datacenter management solution.",
+          "readOnly": true
+        },
+        "datacenterMachineId": {
+          "type": "string",
+          "description": "ID of the machine as tracked by the datacenter management solution.",
+          "readOnly": true
+        },
+        "datacenterManagementServerId": {
+          "type": "string",
+          "description": "ID of the server hosting the datacenter management solution.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of the machine",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "description": "User readable name of the machine as defined by the user in their private datacenter.",
+          "readOnly": true
+        },
+        "megabytesOfMemory": {
+          "type": "number",
+          "format": "double",
+          "description": "Memory in Megabytes.",
+          "readOnly": true
+        },
+        "numberOfCores": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Processor count.",
+          "readOnly": true
+        },
+        "operatingSystem": {
+          "type": "string",
+          "description": "Operating System of the machine.",
+          "readOnly": true
+        },
+        "monthlyBandwidthCost": {
+          "type": "number",
+          "format": "double",
+          "description": "Monthly network cost estimate for the network adapters that are attached to this machine as a group, for a 31-day month.",
+          "readOnly": true
+        },
+        "monthlyStorageCost": {
+          "type": "number",
+          "format": "double",
+          "description": "Monthly storage cost estimate for the disks that are attached to this machine as a group, for a 31-day month.",
+          "readOnly": true
+        },
+        "disks": {
+          "type": "object",
+          "description": "Dictionary of disks attached to the machine. Key is ID of disk. Value is a disk object",
+          "readOnly": true,
+          "properties": {
```

Comment:
>properties [](start = 11, length = 10)

Here too, it is bet not to define stuff inline

Reply:
same as above.